src/components/Layout.tsx

```diff
@@ -23,12 +22,8 @@ export function Layout({ children }: LayoutProps) {
         <Grid item xs={12}>
           <Header />
         </Grid>
-        <Grid item xs={3} height={`calc(100vh - (${style.header.height + style.footer.height}px))`}>
-          <Sidebar />
-        </Grid>
-        <Grid item xs={9}>
-          {children}
-        </Grid>
+        <Sidebar />
```

Comment:
I think you should wrap the Sidebar in a Grid Item to ensure the Height is at least 100vh as by design

Reply:
Nope. Sidebar it's already a Grid component. A 100% height could be an option if accorded to designer. Using an hack with pixels (as made by Anis) isn't the best solution. I think it's rather impossible to have a "short page" that justify the hack